ambuda/views/auth.py

```diff
@@ -79,51 +91,99 @@ def _is_valid_reset_token(row: db.PasswordResetToken, raw_token: str, now=None):
     return True
 
 
+# Copied from https://wtforms.readthedocs.io/en/2.3.x/validators/
+class FieldLength(object):
+    def __init__(self, min=-1, max=-1, message=None):
+        self.min = min
+        self.max = max
+        if not message:
+            message = "Field must be between %i and %i characters long." % (min, max)
```

Comment:
Use f-strings: `f"Field must be between {min} and {max} characters long."` -- note the leading `f`.